org.srb2.SRB2Kart.yaml

```diff
@@ -0,0 +1,50 @@
+app-id: org.srb2.SRB2Kart
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: srb2kart
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --device=all
+  - --persist=.srb2kart
+modules:
+  - shared-modules/glu/glu-9.json
+
+  - name: game-music-emu
+    buildsystem: cmake-ninja
+    cleanup:
+      - /include
+      - /lib/*.so
+      - /lib/pkgconfig
+    sources:
+      - type: archive
+        url: https://bitbucket.org/mpyne/game-music-emu/downloads/game-music-emu-0.6.3.tar.xz
+        sha256: aba34e53ef0ec6a34b58b84e28bf8cfbccee6585cebca25333604c35db3e051d
+
+  - name: srb2kart
+    buildsystem: simple
+    build-commands:
+      - make -C src -j $FLATPAK_BUILDER_N_JOBS LINUX64=1 NOUPX=1 SDL=1
```

Comment:
`LINUX64=1`: I assume this is wrong for `arm`?

I tried to use cmake here, assuming that it would deal with this sort of thing.  Before I figured out how to do cmake, I used `getconf LONG_BIT` to decide whether to use `LINUX=1` or `LINUX64=1`.

Reply:
You are correct. An easy solution would be to set an environment variable based on the architecture and use a substitution. That being said, upstream seems to be using `LINUX` and `NONX86` for their [Debian builds](https://github.com/STJr/SRB2/blob/9d06cb4be459e3ef12f0083a3f21669de3adf8b2/debian-template/rules#L76).

Reply:
I think the substitution option would be similar to what is done in the RPM spec?

Reply:
I did it like this:
https://gitlab.com/cbm755/org-srb2-SRB2Kart/-/commit/f07cde12e782604f6705c736ce60e4bcf7423050

I will try to this in your yml (yml looks much better than json!)

Reply:
What about using `build-options`:

```yaml
build-options:
  arch:
    aarch64:
      env:
        IS64BIT: '64'
    x86_64:
      env:
        IS64BIT: '64'
```

And then using `LINUX${IS64BIT}=1` in the build script?

Reply:
I tried this approach on [SRB2](https://github.com/flathub/org.srb2.SRB2/pull/3) and it worked nicely.

Reply:
oops missed this.  I'll update SRB2Kart to be more like that